cfgov/jinja2/v1/_includes/blocks/feedback.html

```diff
@@ -58,14 +63,17 @@
             </div>
         </div>
     </fieldset>
+    {% else %}
+    <input type="hidden"
+           name="is_helpful"
+           value="{{ request.GET.get('is_helpful', '') }}">
```

Comment:
We want None as a possible value, because the model stores it that way (NullBooleanField) to handle feedback modules with no is_helpful question. So I removed the '' default so that it will pass None instead if the query string is not present.